.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+# The name of the job, which will be displayed for the status check in PR.
+name: README and metadata formatting
+
+# Controls when the action will run.
+# Below triggers the workflow on pull requests to the specified branches.
+on:
+  pull_request:
+    branches:
+      - jen/test-status-checks
+
+# A workflow run is made up of one or more jobs that can run sequentially or
+# in parallel.
+jobs:
+  # This workflow contains a single job called "changes"
+  changes:
```

Comment:
Can we change the name of the job to something more meaningful, e.g. `verify-readme`?

Reply:
Good idea. Changed to `readme-metadata-stylecheck`